elevate-cpanel

```diff
@@ -2983,6 +2985,9 @@ EOS
 
         my ( $major, $minor ) = split( /\./, $mysql_version );
 
+        INFO("Restoring $cnf_file.rpmsave_pre_elevate to $cnf_file...");
```

Comment:
this should move the .rpmsave if the rpmsave was created as a part of the removal?